kview/src/main/java/dev/ikm/komet/kview/controls/KLComponentControl.java

```diff
@@ -109,8 +110,20 @@ public final void setTitle(String value) {
      */
     private final ObjectProperty<EntityProxy> entityProperty = new SimpleObjectProperty<>(this, "entity", null);
     public final ObjectProperty<EntityProxy> entityProperty() { return entityProperty; }
-    public final EntityProxy getEntity() { return entityProperty.get(); }
-    public final void setEntity(EntityProxy value) { entityProperty.set(value); }
+    public final EntityProxy getEntity() {
+        if (((Object)entityProperty.get()) instanceof ConceptRecord) {
```

Comment:
Do you need to down cast to an `Object`?
looks redundant.
New Java pattern matching allows variables.
```java
 if (entityProperty.get() instanceof ConceptRecord myConceptRec) {
     // do work
 } 
```